index.js

```diff
@@ -27,10 +27,19 @@ const getResult = async subprocess => {
 	bufferOutput(subprocess, result, 'stderr');
 
 	try {
-		const [exitCode] = await once(subprocess, 'close');
-		return {...getOutput(result), exitCode};
+		await once(subprocess, 'close');
+		return getOutput(subprocess, result);
```

Comment:
We should throw on non-0 exit code or on signal termination. I will do this in a follow-up PR. (see #8)